src/styles/Book.css

```diff
@@ -0,0 +1,101 @@
+.book {
+  /* border: 1px solid red; */
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  padding: 2rem;
+  background-color: white;
+}
+
+.left {
+  width: 40%;
+}
+
+.cover {
+  display: flex;
+  flex-direction: column;
+  row-gap: 0.5rem;
+  padding: 0 0 2rem 0;
+}
+
+.category {
+  font-size: 1rem;
+  color: gray;
+}
+
+.title {
+  font-size: 2rem;
+  font-weight: bold;
+  color: black;
+}
+
```

Comment:
- Kindly, update the logo, menus, and profile picture positions, minimize the space between the input fields, and  update the font to stick to the given [zepline](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design 👍 
#### your design 👇
<img width="927" alt="2023-07-23_07-35" src="https://github.com/danifromecuador/playbook/assets/52010789/8f107723-78c1-4341-a648-6913dd976362">

#### given Design 👇
<img width="574" alt="2023-07-23_07-32" src="https://github.com/danifromecuador/playbook/assets/52010789/e2794257-2a83-4ef2-85de-cfa355bc6941">